### ISSUE TITLE
Map non-public properties

### ISSUE THREAD

**Issue by bdaniel7** [MapsterMapper/Mapster#170]
I'm trying to map some protected properties, so they are filled with data, but not added in the json.

Is this the correct way? 

```
MapsterAdapter = TypeAdapterConfig<Entity, EntityDto>
				.NewConfig()
				.IncludeMember((member, side) =>
					member.AccessModifier == AccessModifier.Protected
					|| member.AccessModifier == AccessModifier.Public);

```

I don't want to enable it for all mappings, just for some.
I know that [AdaptMember] is working correctly, but I rather keep the dtos clear of attributes.

**Comment by chaowlert** [comment]
So sorry, could you explain more? What is expectation, and what is actual. And pls also provide code to reproduce.

**Comment by bdaniel7** [comment]
I want to map a Dataset to DatasetDto (and DatasetValidity to DatasetValidityDto), using the above mentioned configuration.
The requirement is that ```DatasetValidityPeriods``` is not added in the json generated from the dto.
```
public class DatasetDto
{
    //[AdaptMember]
	protected ICollection<DatasetValidityDto> DatasetValidityPeriods { get; set; }
}
```
When I use [AdaptMember] it works, when I use the TypeAdapterConfig, it doesn't work.

**Comment by chaowlert** [comment]
Hmm, seem like it's bug on `AccessModifier`. For now, please use

```
TypeAdapterConfig<Entity, EntityDto>
				.NewConfig()
				.IncludeMember((member, side) => (model.Info as PropertyInfo)?.GetMethod.IsFamily == true);
```

**Comment by chaowlert** [comment]
fixed in 3.3.2
